comp150-agile/essay.tex

```diff
@@ -0,0 +1,65 @@
+
+When a scrum meeting is performed in the style as suggested by \cite{ DailyScrum, EffectiveScrum}, the description of the barriers they will face is the greatest source of interpersonal growth. The author argues that knowing the process that an individual will do and knowing what they consider barriers the team is able to judge their peer’s strengths and weaknesses. This allows one to promote the strengths and support the weaknesses of their team members. The author argues that this is a deepening of interpersonal relations.
+
+To promote good interpersonal relations there has to be effective communication within the team "The exchange of information is vital to the success of two or more individuals working as a team"\cite [p. 559]{fletcher2006effects}. Within a scrum meeting the scrum master should be a skilled communicator and coordinator. In addition, they can ensure an environment which allows members to "clarify misunderstandings and to acknowledge the receipt of information" \cite [p. 559]{fletcher2006effects} and that individuals feel confident to discuss the checklist (what you did today, what you will do tomorrow, and problems you will face). 
+
+
+\section{Conclusion}
+The author feels that a successful scrum meeting should provide the following; A definition of roles, team bonding, airing of conflicts/impediments, progress of each individual, progress of the project as a whole and ensuring the common goal on track to be achieved (sprint goal).  The current strategy to scrum meetings does not advocate problem solving during the meeting. However, the addition of some methods suggested in  \cite{LeadershipLesson} such as; collective problem solving, if not during the meeting then after could be hugely beneficial to improving the interpersonal relations within the team, research on effective team working supports the authors beliefs. Further research would have to be done to prove this.

```

Comment:
Does *everyone* in the team *need* to be involved? What you are advocating seems inefficient, unless I am missing a point...